x/pocketcore/keeper/claim.go

```diff
@@ -131,6 +131,11 @@ func (k Keeper) ValidateClaim(ctx sdk.Ctx, claim pc.MsgClaim) (err sdk.Error) {
 			return pc.NewOverServiceError(pc.ModuleName)
 		}
 	}
+	// Ensure that the app is not staked to more than the permitted number of chains
+	if pc.ModuleCdc.IsAfterEnforceMaxChainsUpgrade(ctx.BlockHeight()) &&
+		int64(len(app.GetChains())) > k.posKeeper.GetNodeMaxChains(sessionContext) {
```

Comment:
Why do you use the `posKeeper` in the check but the `appKepper` in the error msgt?